### PR TITLE
{DO NOT MERGE}FM-6264 Update rubocop and clean code

### PR DESCRIPTION
**Pull request by tphoney**
Needs review

### CODE REVIEW

File: .sync.yml [hunk 1]
```diff
@@ -6,6 +6,10 @@ appveyor.yml:
   docker_sets:
   - set: docker/ubuntu-14.04
   - set: docker/debian-8
+  extras:
```
Comment (tphoney) [inline .sync.yml]: @DavidS i am unsure of the syntax for extras and .travis does this look sane ?

Comment (DavidS) [inline .sync.yml]: 👍

Comment (tphoney) [inline .sync.yml]: ignore me

Comment (tphoney) [inline .sync.yml]: @HelenCampbell you will want this in your .sync.yml as well

**Pull request closed without merging**